boot/boot.js

```diff
@@ -287,6 +287,42 @@ $tw.utils.decodeURIComponentSafe = function(s) {
 	return v;
 };
 
+
+/*
+Convert a URI List Component encoded string (with `+` as an allowed replacement 
+for `+`) to a string
+*/
+$tw.utils.decodeTWURIList = function(s) {
+	return $tw.utils.decodeURIComponentSafe(
+		s.split('&')
+		.map(function(s) {return s.replaceAll('+', ' ')})
+		.map(function(s) {return s.includes(' ') ? '[[' + s + ']]' : s})
+		.join( ' ')
```

Comment:
Typo - no leading space inside the braces `.join(" ")` -- TW core code uses double quotes for strings.  -- Double quotes also needed in the code above and below for consistency reasons.